jobs/blog_validation_jobs.py

```diff
@@ -29,7 +29,11 @@
 
 from typing import Any, Text # isort:skip # pylint: disable=unused-import
 
-(blog_models,) = models.Registry.import_models( # type: ignore[no-untyped-call]
+MYPY = False
+if MYPY:
+    from mypy_imports import * # pragma: no cover # pylint: disable=import-only-modules,wildcard-import,unused-wildcard-import
```

Comment:
Same thing, here and elsewhere -- no * imports.

Reply:
Ah!! I changed all the files which mere meant to be type annotated by this PR but missed other files which were having mypy issues. Sorry for this!